modules/jax/src/main/ts/ephox/jax/core/HttpError.ts

```diff
@@ -10,10 +12,10 @@ export const enum HttpErrorCode {
   InternalServerError = 500
 }
 
-export interface HttpError {
+export interface HttpError<T extends ResponseType> {
```

Comment:
Not related to this change, but if you don't mind can we please add `readonly` to these 3 properties as well while we're making changes?

Reply:
Done.